### PR TITLE
appveyor: disable test 571 in two cmake builds

### PR DESCRIPTION
**Pull request by bagder**
...  they're simply too flaky there.

### CODE REVIEW

**Comment by MarcelRaad** [comment]
Why not mark them as flaky? Is there anything special to these two build configurations?

**Comment by bagder** [comment]
Only because I don't want to disable them for every build everywhere.

Ideally we introduce a way in the future to give us more qualifiers to "flaky". Like platforms, build combinations etc.

**Comment by dfandrich** [comment]
You could start now. Just mark this test as flaky-cmake and add !flaky-cmake to
the test keywords it uses. The mechanism can always be tweaked later.

**Comment by mback2k** [comment]
For this I introduced the `~` prefix for test numbers in `runtests.pl`. It does not disable them, but ignores the result in the CI status.

**Comment by mback2k** [comment]
Also this seems to be the AppVeyor equivalent of https://github.com/curl/curl/pull/5954. Unfortunately I wasn't able to debug this RTSP test on Windows (yet).

File: appveyor.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@ environment:
         HTTP_ONLY: OFF
         TESTING: ON
         SHARED: OFF
-        DISABLED_TESTS: "!1139 !1501"
+        DISABLED_TESTS: "!1139 !1501 !571"
```
Comment (mback2k) [inline appveyor.yml:90]: Please sort the test numbers for consistency. :-)

**Comment by bagder** [comment]
> Just mark this test as flaky-cmake

But its cmake-on-appveyor, or possibly cmake-on-win32. I think I rather stick to disabling it in appveyor for now.

**Comment by dfandrich** [comment]
Yeah, if it's that specific then that's probably best.

**Comment by mback2k** [comment]
Actually it is either not working due to MinGW or MSYS since it is also broken for the autotools based builds on Azure Pipelines. So I don't think this is caused by CMake.

**Comment by bagder** [comment]
Short term, we can keep adding test numbers to ignore in the CI setup files, and then we can go over them all and see which are ignored where and see if we can draw conclusions and/or make better systems to handle them.

**Pull request closed without merging**